lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -154,6 +160,26 @@ module Constants
           'Google::Apis::LoggingV2beta1::LogEntryOperation'
         ]
       }
+
+      # Used as a map key for entries with no locally unique id in the payload.
+      NO_UNIQUE_ID_PRESENT = 'no_locally_unique_id'
+
+      # Metadata Agent support.
+
+      # Use empty string as request path when locally-unique key of monitored
+      # resource can be implicitly inferred by Metadata Agent.
+      IMPLICIT_MONITORED_RESOURCE_UNIQUE_KEY = ''
+
+      DEFAULT_METADATA_AGENT_URL =
+        'http://local-metadata-agent.stackdriver.com:8000'
+
+      # The label name of locally unique id in the json payload. When a record
+      # has this field in the payload, we will use the value to retrieve
+      # monitored resource from Stackdriver Metadata agent.
+      DEFAULT_UNIQUE_ID_KEY = "#{DEFAULT_PAYLOAD_KEY_PREFIX}/locally_unique_id"
```

Comment:
This would probably problematic to change at this point, but I just realized that this string does not mention the word "resource" anywhere. It would make more sense to call this "#{DEFAULT_PAYLOAD_KEY_PREFIX}/local_resource_id" instead. WDYT?

Reply:
I think `local_resource_id` makes more sense. And it's not too late to change it at this point. So I just changed it along with all the comments. I'll change the configs in the EAP doc once I push a new gem.